common/src/fakeData/fakeSkillFamily.ts

```diff
@@ -0,0 +1,31 @@
+import faker from "faker";
+import { SkillCategory, SkillFamily, Skill } from "../api/generated";
+
+const generateSkillFamily = (name: string) => {
+  return {
+    category: faker.random.arrayElement([
+      SkillCategory.Behavioural,
+      SkillCategory.Technical,
+    ]),
+    description: {
+      en: faker.lorem.sentences(),
+      fr: faker.lorem.sentences(),
+    },
+    id: faker.datatype.uuid(),
+    key: name,
+    name: {
+      en: name,
+      fr: name,
```

Comment:
We often like to prefix our localized fake data, just so you can tell it's right if you switch languages. Something like ``{en: `EN ${name}`, fr: `FR ${name}`}``. This would go for the description as well, where you can prefix the lorem sentence.

Reply:
Sounds good, I knew I should probably make them look different but I hadn't considered doing that.